cookbook/routing/index.rst

```diff
@@ -9,6 +9,8 @@ Routing
     redirect_in_config
     method_parameters
     service_container_parameters
-    custom_route_loader
     redirect_trailing_slash
     extra_information
+    extending_routing
+        custom_route_loader
+        multiple_routers
```

Comment:
there is no /cookbook/routing/map.rst.inc file i think. do you have an idea how we should do this? or should we give up on making this a sub-group and just cram it into one article? or a series that repeat part of the title: "Extending Routing: ..."?